web/src/routes/(user)/archive/+page.svelte

```diff
@@ -37,7 +38,7 @@
       <AddToAlbum />
       <AddToAlbum shared />
     </AssetSelectContextMenu>
-    <DeleteAssets onAssetDelete={(assetId) => assetStore.removeAsset(assetId)} />
+    <DeleteAssets force={!$featureFlags.recycleBin} onAssetDelete={(assetId) => assetStore.removeAsset(assetId)} />
```

Comment:
I think the idea was to have `force` here only on the trash screen. We might want to remove all the rest of these feature flag usages and put them into the `DeleteAssets` svelte file directly. Ie default to the behavior of the feature flag inside of `DeleteAssets` directly, and have an option to always permanently delete by using `force` on the trash screen.

Reply:
Updated!